CHANGELOG.rst

```diff
@@ -1,10 +1,30 @@
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Changelog for package mara
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-0.0.1 (2019-02-28)
-------------------
-* Adapted mara_gazebo_plugin to HRIM coliza
-* Moved Robotiq modular grippers to a specific repository https://github.com/AcutronicRobotics/robotiq_modular_gripper
-* Moved Hans modular joints to a specific repository https://github.com/AcutronicRobotics/hans_modular_servomotor
-* Contributors: Alejandro Hernandez, Igor Recio, Lander Usategui San Juan, Nestor Gonzalez
+**************
+MARA changelog
+**************
+
+crystal-v1.0.0 (2019-06-12)
+###########################
+
+**Added**
+
+* Whole infraestructure base on HRIM Coliza
+* Compilance with https://github.com/AcutronicRobotics/robotiq_modular_gripper/tree/crystal
+* Models for Reinforcement Learning
+* Two MARA robots
+* RViz2 visualization
+
+**Changed**
+
+* Measurement values in the URDFs
+* Instructions in readme
+* Installation of additional ROS2 dependencies trough Debian packages, instead of from source
+* Topic names
+* Files (unification)
+
+**Removed**
+
+* Unused/Deprecated files
```

Comment:
also here if there are any files we no longer use just to inform users of this repo